style-guide/about-style-guide.md

```diff
@@ -0,0 +1,165 @@
+project documentation. The overall goal of a style guide is to ensure quality
+and consistency throughout the project’s documentation, which is especially
+important if different authors are contributing to the documentation over time.
+
+A style guide can provide early guidance to new contributors as they make
+decisions related to your project’s documentation. Contributors often have to
+make subjective decisions about the documentation, such as:
+
+- Which style guide your project will defer to as its default style guide
+- How to refer to terms that are unique or specific to that project
+- How to format certain types of content
+- What tone, attitude, or level of formality you recommend using in the project
+  (usually conveyed through word choice)
+- Any specific rules you want to highlight for documentation contributors, such
+  as language usage disputes that seem to arise periodically within the
+  community
+
+A style guide provides your project’s answers to these questions and ensures
+that all of the contributors are working from the same common understanding.
+
+## How to make a style guide scale with your project
+
+A project’s style guide usually starts out as a stub at the beginning of a
+project. For example, it might simply indicate which style guide is your
+project’s default guide, such as the Google Developer’s style guide or the
+Microsoft Manual of Style. It might link to specific elements from the default
+guide that are most relevant to your project or point out any exceptions to the
+default guide.
+
+As your project grows and matures, your style guide can grow along with it. For
+example, debates can (and will) arise about how to refer to a specific term in
+the documentation, such as whether to capitalize or hyphenate a term that is
+unique to the project. Once resolved, you can capture decisions in the style
+guide. Then, if that debate arises again, you can simply refer contributors to
+your style guide rather than rehashing the same debate every time a new
+contributor joins the project.
+
+However, keep in mind that some teams consistently struggle to maintain a style
+guide over time. The guiding principle here is that a small amount of work up
+front can save a lot more work later. It’s much easier to set your standards in
+a style guide early in your project’s life cycle than it is to retroactively
+clean up your documentation later when the project is more mature. With small
+but regular updates, your style guide can evolve alongside your project. It will
+help your contributors get up to speed quickly and start writing high quality
+documentation.
+
+To help your project scale, consider setting a revolving meeting or agenda item
+to review the style guide and address style guide issues on a periodic basis
+(such as once a month). Perhaps you could make someone on the team the project
+owner for the style guide to ensure someone has the responsibility to update the
+guide after style guide reviews.
+
```

Comment:
I think you should add a section titled "Target audience"
In it you describe that your audience for a style guide are the people who write your documentation. You will know them best. They typically will include software engineers, product mangers, tech writers if you are lucky enough to have them on the team, ...

Reply:
Good suggestion. I've added a section for "Intended audience and scope" to both the template and the template description.